ThunderRequest/TSCRequestController.m

```diff
@@ -160,7 +160,13 @@ - (nonnull TSCRequest *)get:(nonnull NSString *)path withURLParamDictionary:(nul
     request.requestHTTPMethod = TSCRequestHTTPMethodGET;
     request.path = path;
     request.URLParameterDictionary = URLParamDictionary;
-    request.requestHeaders = self.sharedRequestHeaders;
+	
+	NSMutableDictionary *requestHeaders = [self.sharedRequestHeaders mutableCopy];
+	// In some API's an error will be returned if you set a Content-Type header
+	// but don't pass a body (In the case of a GET request you never pass a body)
+	// so for GET requests we nill this out
+	[requestHeaders removeObjectForKey:@"Content-Type"];
```

Comment:
What's going on here? Why is this indented?

Reply:
Ohh god... It seems like the whole file is indented with spaces 😑